RecoEgamma/EgammaHLTProducers/plugins/HLTCaloObjInRegionsProducer.cc

```diff
@@ -318,4 +315,35 @@ void EtaPhiRegionData<CandCollType>::getEtaPhiRegions(const edm::Event& event,
   }
 }
 
-#endif
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/HcalDigi/interface/HBHEDataFrame.h"
+using HLTHcalDigisInRegionsProducer = HLTCaloObjInRegionsProducer<HBHEDataFrame>;
+DEFINE_FWK_MODULE(HLTHcalDigisInRegionsProducer);
+
+#include "DataFormats/HcalDigi/interface/QIE11DataFrame.h"
+#include "DataFormats/HcalDigi/interface/HcalDigiCollections.h"
+using HLTHcalQIE11DigisInRegionsProducer = HLTCaloObjInRegionsProducer<QIE11DataFrame, QIE11DigiCollection>;
+DEFINE_FWK_MODULE(HLTHcalQIE11DigisInRegionsProducer);
+
+#include "DataFormats/HcalDigi/interface/QIE10DataFrame.h"
+#include "DataFormats/HcalDigi/interface/HcalDigiCollections.h"
```

Comment:
This is a duplicate of L325.
I understand that this could be left here for symmetry, and I have no strong preference for it: I let you decide whether to remove or not